source/addins/ProAppDistanceAndDirectionModule/ProAppDistanceAndDirectionModule/ViewModels/ProTabBaseViewModel.cs

```diff
@@ -1349,6 +1351,23 @@ protected FeatureLayer GetFeatureLayerByNameInActiveView(string featureLayerName
             return viewLayer;
         }
 
+        internal static async void RemoveSpatialIndexOfLayer(string layerName)
```

Comment:
If this was a problem with the .lyrx layer package file having a spatial index - we could have run this tool prior to packaging the file to address this issue also